00_introduction/21_architecture_nodes.txt

```diff
@@ -33,18 +33,26 @@ available for these platforms:
 
 * Ubuntu 12.10 (quantal)
 
+* IBM AIX 5.3
+
+* IBM AIX 6.1
+
+* IBM AIX 7.1
+
 [TIP]
 
 [[install-on-windows, Install on Microsoft Windows]] 
 
-.Windows Nodes 
+.Windows and AIX Nodes
 
 ====
 
-Installing Rudder on Windows requires the commercial version of CFEngine (named Nova).
+* On Windows, installing Rudder requires the commercial version of CFEngine (named Nova)
+* On IBM AIX, installing Rudder requires a commercial version of the Rudder agent
```

Comment:
Well. This is not strictly true, the AIX agent is open source. Normation just does not provide built packages of it.

Could you rephrase this to explain that to get a RPM package for AIX you need to contact Normation to obtain a demo version etc?